### ISSUE TITLE
UserWarning: CUDA initialization: CUDA driver initialization failed, you might not have a CUDA gpu

### ISSUE THREAD

**Issue by srdfjy** [wenet-e2e/wenet#700]
Hi，I have a usable gpu card number 3, when I run the default aishell s0 example, the following error occurred.

nvidia-smi info： NVIDIA-SMI 440.64.00    Driver Version: 440.64.00    CUDA Version: 10.2

error
----------------------------------------------------------------------------------
the number of model params: 49349138
/data/home/miniconda3/envs/wenet/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: CUDA driver initialization failed, you might not have a CUDA gpu. (Triggered internally at  /opt/conda/conda-bld/pytorch_1631630839582/work/c10/cuda/CUDAFunctions.cpp:115.)
  return torch._C._cuda_getDeviceCount() > 0
2021-10-21 14:32:41,099 INFO Checkpoint: save to checkpoint exp/conformer/init.pt
2021-10-21 14:32:41,453 INFO Epoch 0 TRAIN info lr 8e-08
2021-10-21 14:32:41,456 INFO using accumulate grad, new batch size is 4 timeslarger than before
2021-10-21 14:32:47,093 DEBUG TRAIN Batch 0/7507 loss 223.928131 loss_att 83.471237 loss_ctc 551.660828 lr 0.00000008 rank 0
2021-10-21 14:37:10,424 DEBUG TRAIN Batch 100/7507 loss 232.583084 loss_att 83.298691 loss_ctc 580.913330 lr 0.00000208 rank 0

**Comment by robin1001** [comment]
It's related to your cuda driver or pytorch, Please first make sure cuda is available in your pytorch.
